packages/emails/src/components/CallToActionIcon.tsx

```diff
@@ -19,7 +11,6 @@ export const CallToActionIcon = ({
       height: "1rem",
       width: "1rem",
       marginLeft: "0.5rem",
-      ...(secondary && { filter: "brightness(50%)" }),
```

Comment:
filter property doesn't work in emails https://github.com/calcom/cal.com/issues/8890#issuecomment-1556293989